scene/gui/color_picker.cpp

```diff
@@ -1246,11 +1308,11 @@ void ColorPicker::_hsv_draw(int p_which, Control *c) {
 			Vector<Point2> points;
 			Vector<Color> colors;
 			Color col;
-			col.set_ok_hsl(ok_hsl_h, ok_hsl_s, 1);
+			col.set_ok_hsl(h, s, 1);
 			Color col2;
-			col2.set_ok_hsl(ok_hsl_h, ok_hsl_s, 0.5);
+			col2.set_ok_hsl(h, s, 0.5);
 			Color col3;
-			col3.set_ok_hsl(ok_hsl_h, ok_hsl_s, 0);
+			col3.set_ok_hsl(h, s, 0);
```

Comment:
That doesn't look correct.

Reply:
It must have been merging artifact, it looks like I accidentally missed those changes when resolving a merge conflict from master and forced the old version. Great that you found it, it's fixed now.